scripts/test-backstop

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Install mimimist for use in Backstop test scripts
+cd ./backstop-tests/ && npm init -f && npm install minimist@1.2.0
+docker run --rm -v "$(pwd)":/src backstopjs/backstopjs test --config=backstop.js
```

Comment:
I had to add the "/" again, in front of the "$" to get this to work, probably a Windows issue. If this also works for you with "/$" I suggest that's how it should be.